mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -261,12 +274,18 @@ CLASS none/sg net/minecraft/entity/Entity
 		ARG 0 index
 	METHOD l setBreath (I)V
 		ARG 0 value
+	METHOD m startRiding (Lnone/sg;)Z
+		ARG 0 entity
+	METHOD n canStartRiding (Lnone/sg;)Z
+		ARG 0 entity
 	METHOD n_ hasCustomName ()Z
-	METHOD o startRiding (Lnone/sg;)V
+	METHOD o addPassenger (Lnone/sg;)V
```

Comment:
Passenger or rider would both make sense, with all the ride stuff flying around I think a little bit of separation would make it easier to work with.